### ISSUE TITLE
Special characters in testcase/step name prevents Sakuli to finish execution

### ISSUE THREAD

**Issue by cliffle** [sakuli/sakuli#470]
**Version**
2.3.1-next.278

**Short overview**
According to the logs, Sakuli does not finish execution when the test case name or the step name contains a special character. Thus no results are forwarded to e.g. Prometheus.

**Detailed error description**

When executing the test below, the following things are printed to `sakuli.log`

```
[Thu Aug 20 2020 08:29:57 GMT+0200 (Central European Summer Time)] INFO: Loaded Sakuli with prometheusPreset,legacyPreset
[Thu Aug 20 2020 08:29:57 GMT+0200 (Central European Summer Time)] DEBUG: Prometheus Forwarder Configuration:
enabled: true,
apiHost: localhost,
apiPort: 9091,
apiJob: pushgateway
[Thu Aug 20 2020 08:29:57 GMT+0200 (Central European Summer Time)] INFO: Started Execution
[Thu Aug 20 2020 08:29:58 GMT+0200 (Central European Summer Time)] INFO: Started Testsuite integrationtest
[Thu Aug 20 2020 08:29:58 GMT+0200 (Central European Summer Time)] INFO: Started Testcase Testcase Ä
[Thu Aug 20 2020 08:29:58 GMT+0200 (Central European Summer Time)] INFO: Started Step UNNAMED
[Thu Aug 20 2020 08:29:58 GMT+0200 (Central European Summer Time)] TRACE: Start action navigateTo
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: Finish action navigateTo successfully after 1.851s
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: Start action highlight
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: 57 Elements found after applying relations 
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] DEBUG: Converted string "EXPLORE" to regex /^s*EXPLOREs*$/
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] DEBUG: Found 57 Elements for RegEx: /^s*EXPLOREs*$/
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: 57 Elements found after applying identifier EXPLORE
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: scroll into view started with element: 
tag: a,
id: ,
class: btn btn-transparent,
text: EXPLORE
[Thu Aug 20 2020 08:30:00 GMT+0200 (Central European Summer Time)] TRACE: scroll into view finished, page did scroll
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] TRACE: Finish action highlight successfully after 2.795s
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] INFO: Finished Step Step 1 with state Ok 4.649s
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] INFO: Started Step UNNAMED
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] INFO: Finished Step UNNAMED with state Ok 0s
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] INFO: Finished Testcase Testcase Ä with state Ok 4.653s
[Thu Aug 20 2020 08:30:02 GMT+0200 (Central European Summer Time)] INFO: Finished Testsuite integrationtest with state Ok 4.705s
[Thu Aug 20 2020 08:30:03 GMT+0200 (Central European Summer Time)] DEBUG: Closed webdriver
```

`INFO: Finished Execution` and `INFO: Forwarding check result to Prometheus.` are missing.
Even without the forwarder configured, `Closed webdriver` is the last log message.

**Steps to reproduce error**

```javascript
(async () => {
  const testCase = new TestCase("Testcase Ä");
  try {
    await _navigateTo("https://sakuli.io");
    await _highlight(_link("EXPLORE"));
    await testCase.endOfStep("Step 1")
  } catch (e) {
    await testCase.handleException(e);
  } finally {
    await testCase.saveResult();
  }
})();
```

**Additional content**
https://github.com/sakuli/sakuli-enterprise-forwarder/issues/68

**Comment by cliffle** [comment]
It seems that the import for the `WriteableStream` in `createWriteStreamConsumer` is wrong 
https://github.com/sakuli/sakuli/blob/c19fb226daca89300d2035e4b95aeb105618b462/packages/sakuli-commons/src/logger/consumer/create-write-stream-consumer.function.ts#L4

We create our `WritableStream` from `fs`
https://github.com/sakuli/sakuli/blob/c19fb226daca89300d2035e4b95aeb105618b462/packages/sakuli-commons/src/logger/consumer/create-file-log-consumer.function.ts#L4

**Comment by svettwer** [comment]
might be related to https://github.com/sakuli/sakuli/issues/376

**Comment by cliffle** [comment]
Closed in favor of #376
